src/main/scala/com/raphtory/core/deploy/Raphtory.scala

```diff
@@ -23,6 +23,53 @@ import org.apache.pulsar.client.api.Schema
 import scala.reflect.ClassTag
 import scala.reflect.runtime.universe._
 
+/**
+  * {s}`Raphtory`
+  *  : `Raphtory` object for creating Raphtory Components
+  *
+  * ## Methods
+  *
+  *   {s}`createGraph(spout: Spout[T] = new IdentitySpout[T](), graphBuilder: GraphBuilder[T], customConfig: Map[String, Any] = Map()): RaphtoryGraph[T]`
+  *    : Create Graph using spout, graph-builder and custom config
```

Comment:
What kind of object does this return? what can we do with that object? 
What goes in the custom config are there any examples? what can we do with this client object that this creates?